src/lib/util.ts

```diff
@@ -79,3 +79,16 @@ export function typedEntries<K extends string, V>(o: Record<K, V>): [K, V][] {
 export function typedKeys<K extends string>(o: Record<K, unknown>): K[] {
     return Object.keys(o) as K[];
 }
+
+export function getColorMode(numberOfChannels: number | string) {
```

Comment:
1. Parsing numbers should not be the responsibility of this function.
2. You only ever call the function with numbers.
```suggestion
export function getColorMode(numberOfChannels: number) {
```